deploy/csi-azuredisk-node.yaml

```diff
@@ -91,6 +91,7 @@ spec:
             - "--metrics-address=0.0.0.0:29605"
             - "--enable-perf-optimization=true"
             - "--allow-empty-cloud-config=true"
+            - "--get-node-info-from-labels=false"
```

Comment:
mmm I thought these files were generated by running the chart against values and dumping them here, it looks like they're updated manually because this file doesn't have the other flags in the chart.

if we want to deploy a release which files should we use?

- the files in `deploy/<release>`? if so, are these files the ones tested?
- the result of applying some values to the chart from the tag <release>? if so, are these files the ones tested?

Reply:
there are two ways to install this driver:
 - kubectl apply `deploy/<release>`
 - helm chart install
pls always use helm chart install for customization: https://github.com/kubernetes-sigs/azuredisk-csi-driver/tree/master/charts#v1-parameters

Reply:
understood, thanks